pkg/ccl/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -280,6 +280,14 @@ func createLogicalReplicationStreamPlanHook(
 		if cr, ok := options.GetDefaultFunction(); ok {
 			defaultConflictResolution = *cr
 		}
+
+		var jobID jobspb.JobID
```

Comment:
nit: I find deferred control flow hard to follow when reading code. This can be written more clearly using standard error handling.

var jobID jobspb.JobID
err := p.ExecCfg(... {

}
if err != nil {
   return err
}
resultCh <- tree.Datums{tree.NewDInt(tree.DInt(jobID))}
return nil

Reply:
Done.